samples/financial_transcripts/generate_benchmark_qa.ipynb

```diff
@@ -0,0 +1,562 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'MSFT'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Ticker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Generate Questions (using Azure OpenAI only)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1. What was the percentage increase in server products and cloud services revenue for MSFT in Q1 FY23?\n",
+      "Answer: The server products and cloud services revenue increased by 22% in constant currency for MSFT in Q1 FY23.\n",
+      "\n",
+      "2. How much did Azure and other cloud services revenue grow in constant currency for MSFT in Q1 FY23?\n",
+      "Answer: Azure and other cloud services revenue grew 35% in constant currency for MSFT in Q1 FY23.\n",
+      "\n",
+      "3. Did Azure consumption growth meet expectations for MSFT in Q1 FY23?\n",
+      "Answer: No, Azure consumption growth was about a point lower than expected for MSFT in Q1 FY23.\n",
+      "\n",
+      "4. What was the impact on the enterprise mobility and security installed base for MSFT in Q1 FY23?\n",
+      "Answer: The enterprise mobility and security installed base grew by 18% to over 232 million seats for MSFT in Q1 FY23.\n",
+      "\n",
+      "5. How did the revenue for MSFT's on-premises server business perform in Q1 FY23?\n",
+      "Answer: The revenue was flat and increased 4% for MSFT's on-premises server business in Q1 FY23.\n",
+      "\n",
+      "6. What was the reason for the decrease in MSFT's margins in Q1 FY23?\n",
+      "Answer: MSFT elected not to throttle back OpEx despite the weakness on the Windows side, which caused margins to drift a little bit lower in Q1 FY23.\n",
+      "\n",
+      "7. Why did MSFT decide not to throttle back OpEx in Q1 FY23?\n",
+      "Answer: MSFT wanted to continue to invest where they were seeing substantial growth, and they believe it's important to continue investing in those areas.\n",
+      "\n",
+      "8. What was the reason for the moderation in Azure consumption growth in Q1 FY23?\n",
+      "Answer: MSFT was helping customers optimize current workloads while they prioritize new workloads, which caused moderation in Azure consumption growth in Q1 FY23.\n",
+      "\n",
+      "9. Did MSFT experience a sales mix shift in Q1 FY23?\n",
+      "Answer: Yes, MSFT experienced a little bit more sales mix in Q1 FY23, given the weakness on the Windows side.\n",
+      "\n",
+      "10. Did MSFT meet their three-months-ago guidance for flat margins in Q1 FY23?\n",
```

Comment:
i'm not an SME, but both the question and the answer look wrong to me :)

here's what i found in the text: "It looks like you’ve elected not to throttle back OpEx to enable you to meet the three-months-ago guidance for flat margins, so they’ll now drift a little bit lower. Do you mind talking through that decisions? Why not throttle back OpEx as a counter to that Windows pressure? "